src/software/ai/evaluation/calc_best_shot.cpp

```diff
@@ -9,23 +9,29 @@ namespace Evaluation
                                            const std::vector<Point> &obstacles,
                                            double radius)
     {
-        // Use angleSweepCircle function to get the pair
-        return angleSweepCircles(p, goal_post_neg, goal_post_pos, obstacles, radius);
+        // Use shot evaluation function to get the best Shot
```

Comment:
from code reading, this is just a helper function right? Slightly out of scope, but can we rename it and make it private?

Reply:
I'll throw it in an anon namespace :+1:

Reply:
Actually just realized I can't test the functions if it is anonymous.
@MathewMacDougall @garethellis0 What do you think?

Reply:
It technically is being used as a helper but is still valid on its own. I'd say leave it public.

ie. I consider the other functions that build on this "convenient wrappers", rather than this being a "helper" of those functions.

Reply:
yea public is fine but renaming would be appreciated